fpgaperf.py

```diff
@@ -817,9 +823,11 @@ def list_seedable():
     for t in get_seedable():
         print(t)
 
-def check_env():
+def check_env(to_check):
```

Comment:
if to_check is invalid, does this silently fail?

Reply:
Fixed.

Reply:
Fixed.